mscore/resourceManager.cpp

```diff
@@ -85,13 +122,14 @@ void ResourceManager::displayExtensions()
             }
       int rowCount = result.object().keys().size();
       rowCount -= 2; //version and type
+      extensionsTable->clearSpans();
       extensionsTable->setRowCount(rowCount);
 
       int row = 0;
       int col = 0;
       QPushButton* buttonInstall;
       QPushButton* buttonUninstall;
-      extensionsTable->verticalHeader()->show();
```

Comment:
I don't know what this line of code's effect has been, since the `extensionsTable` has never shown a vertical header(i.e. the row count number in the left of each row) even if we call `show()`, due to some unknown reason...

But after this commit, the line of code begins to function somehow, and we get the following different appearance of the extensions table, with the vertical header:
![show_applied](https://user-images.githubusercontent.com/28099570/61355830-bc14df00-a8a7-11e9-8bfc-6b4739a6fab9.png)

I don't know if the change is wanted. However, in order to keep UI the same as before, I deleted that line for now.

Reply:
Maybe just comment it out instead? Not sure...